lib/db/access.php

```diff
@@ -55,6 +55,11 @@
 defined('MOODLE_INTERNAL') || die();
 
 $capabilities = array(
+    'moodle/urlpreview:usetool' => array(
```

Comment:
I think we need to split this capability into two. One is the capability for using the debugging tool which will be default site admin / course editors and stay with the tool, the other will be the one end users like students or even guests use and I think it should default to any user, even a bit logged in / guest user. Eg if you are not logged in and you view a forum you should still get the url preview same as everyone else